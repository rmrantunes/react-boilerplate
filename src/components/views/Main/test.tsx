import { screen, render } from '@testing-library/react'
import { Main } from '.'

describe('<Main />', () => {
  it('Should render heading', async () => {
    render(<Main />)

    const heading = await screen.findByRole('heading', {
      name: /react avanÃ§ado/i
    })
    expect(heading).toBeInTheDocument()
    expect(heading).toMatchSnapshot()
  })

  it('should render bg color correctly', async () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveStyle({
      'background-color': '#06092b'
    })
  })
})
