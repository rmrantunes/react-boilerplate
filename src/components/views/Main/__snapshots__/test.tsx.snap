// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render heading 1`] = `
.c0 {
  font-size: 2.5rem;
}

<h1
  class="c0"
>
  React avan√ßado
</h1>
`
